.circleci/config.yml

```diff
@@ -6,37 +9,20 @@ defaults:
       CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
       BUNDLE_PATH: vendor/bundle
     macos:
-      xcode: "10.0.0"
+      xcode: "11.4.1"
     shell: /bin/bash --login -eo pipefail
   - &linux-config
     environment:
       CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
       CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
       BUNDLE_PATH: vendor/bundle
     docker:
-      - image: norionomura/jazzy:swift-4.2.0
+      - image: marekfort/docker-stencilswiftkit:swift-5.2
```

Comment:
The only nit I'd make here is that I think it's a missed opportunity to have a more generic Docker image name — after all this one could be used by any other project that might require just the same ruby version, homebrew, and similar tools… but 🤷 that's not a big deal and could be kept for a later migration if we really want to.

Reply:
Tadaaa: https://hub.docker.com/repository/docker/swiftgen/swift